### ISSUE TITLE
digdag scheduler need more useful error message.

### ISSUE THREAD

**Issue by hiroyuki-sato** [treasure-data/digdag#130]
From [twitter](https://twitter.com/nuna/status/743711225402400768)

``digdag scheduler`` need more useful message.

```
% digdag init foo
2016-06-18 10:09:58 +0900: Digdag v0.8.1
  Creating foo/.gitignore
  Creating foo/tasks/shell_sample.sh
  Creating foo/tasks/repeat_hello.sh
  Creating foo/tasks/__init__.py
  Creating foo/foo.dig
Done. Type `cd foo` and then `digdag run foo.dig` to run the workflow. Enjoy!
```

```
% cd foo/
% digdag scheduler 
2016-06-18 10:10:24 +0900: Digdag v0.8.1
error: 
```

**Comment by hiroyuki-sato** [comment]
```
digdag sched [options...]
  Options:
        --project DIR                use this directory as the project directory (default: current directory)
```

https://github.com/treasure-data/digdag/blob/master/digdag-cli/src/main/java/io/digdag/cli/Sched.java#L43

Is the default value "."?

```c
    String projectDirName = null;
```

**Comment by frsyuki** [comment]
Fixed at ca922c14d9587efc29e2bc904296f0fd9884c6e5

**Comment by frsyuki** [comment]
Error also fixed at fe9adb5b6858ce5c97a3a8a11f8dcecc5b544437

**Comment by hiroyuki-sato** [comment]
Thanks!. It's worked as I expected.

```yaml
timezone: "Asia/Tokyo"

schedule:
  minutes_interval>: 1

+current_date:
  sh>: echo `date`

+echo_hello:
  sh>: echo "hello"
```

```
digdag-0.8.2-SNAPSHOT.jar scheduler
2016-06-21 08:47:52 +0900: Digdag v0.8.2-SNAPSHOT
2016-06-21 08:47:53 +0900 [INFO] (main): Added new revision 1
2016-06-21 08:47:53 +0900 [INFO] (main): Starting server on 127.0.0.1:65432
2016-06-21 08:47:53 +0900 [INFO] (main): XNIO version 3.3.3.Final
2016-06-21 08:47:54 +0900 [INFO] (main): XNIO NIO Implementation Version 3.3.3.Final
2016-06-21 08:48:00 +0900 [INFO] (scheduler-0): Starting a new session project id=1 workflow name=foo session_time=2016-06-21T08:48:00+09:00
2016-06-21 08:48:01 +0900 [INFO] (0027@+foo+current_date): sh>: echo `date`
2016年 6月21日 火曜日 08時48分01秒 JST
2016-06-21 08:48:01 +0900 [INFO] (0027@+foo+echo_hello): sh>: echo "hello"
hello
```
